documentation/src/main/markdown/packagemgrs/python.md

```diff
@@ -6,87 +6,94 @@
 
 ## Overview
 
-[solution_name] has three detectors for Python:
-
-* Pip detector
-* Pipenv detector
-* Pipfile lock detector
+[solution_name] detectors for discovery of dependencies in Python:
+
+* PIPENV detectors
+	* Pipenv detector
+	* Pipfile lock detector
+* PIP detectors
+	* Pip Native Inspector
+	* Pip Requirements File Parser
 * Poetry detector
 
-## The Pip detector
+## PIPENV Detectors
+
+## Pipenv detector
+
+The Pipenv detector attempts to run on your project if either a Pipfile or Pipfile.lock file is found.
+
+The Pipenv detector requires Python and Pipenv executables.
+
+* [solution_name] looks for python (or python3 if the python3 property is set to true) on $PATH. You can override this by setting the python path property.
```

Comment:
The python3 property was deprecated and removed in 7.0.0. The phrase in the parenthesis can be removed.

Reply:
Done. Thank you.